### ISSUE TITLE
Change opportune parry to trigger on enemy attack instead of hit.

### ISSUE THREAD

**Issue by CAWilliam** [Truinto/DarkCodex#156]
Hey there. I was really enamoured with the idea of your opportune parry feat and built a character around it. But I found that in practice it doesn't really pop off like I was hoping because you only make parry attempts on attacks that would succesfully hit you. 

The aldori defender and duelist paths are inherently quite defensive, so what I experienced was not the "shredding weak opponents that try to attack you" that I was hoping for. Since many enemies were only hitting me with high rolls this also made it much less likely for me to succesfully parry even if I got a parry attempt. Often ending my turns surrounded by enemies but still with several AoO's remaining due to lack of valid parry attempts.

In short, I'd love it if this could be an option so we can live the supreme duelist who ripostes hordes to death fantasy.

**Comment by Truinto** [comment]
Please try out "Patch: Parry Always"

**Comment by CAWilliam** [comment]
Just gave this a shot on the new 1.5.2 version but it didn't seem to be working. Though I won't discount that another mod could be interfering. 
![image](https://user-images.githubusercontent.com/73697563/196805069-f55ec939-8642-4082-a502-fe289e23784d.png)

**Comment by Truinto** [comment]
You did enable it and restart the game?

**Comment by CAWilliam** [comment]
I did, aye.

**Comment by Jorill** [comment]
I just did some testing and I can't get it to work either. Sometimes I will get a parry within a round, but it appears to be working on vanilla conditions (not very often, no more than 1x/round).

I made sure I had it enabled/restarted game. Then made sure the character was using a 1h/light weapon (rapier) and had both duelist parry and duelist opportune parry. The character had combat reflexes, but just to be sure, I used toybox to add 100 attacks of opportunity as well.

This is great idea and hope it is doable. Swashbuckler was always such a fun class and I was excited to see this added. I definitely miss it from the kingmaker mod.

**Comment by Truinto** [comment]
Sounds more like that feature doesn't work at all. Fine, will check when I find time.

**Comment by Truinto** [comment]
Probably works now.

**Comment by Jorill** [comment]
This is awesome, works great!
